### ISSUE TITLE
Bulbs not found for Homebridge

### ISSUE THREAD

**Issue by mweintr** [kpsuperplane/homebridge-wiz-lan#14]
I installed v2.0.5.

Running Homebridge 1.1.1 (Node.js version 12.15.0) and I can't add my two Wiz bulbs to Homekit via this plugin. My config has this:
`        {
            "platform": "WizSmarthome",
            "name": "WizSmarthome"
        },`

**Comment by mitch7391** [comment]
I don’t think I was able to find them until I had first set them up in the Wiz app. Not sure if you have tried that?

**Comment by kpsuperplane** [comment]
Agreed with @mitch7391, unfortunately I'm not quite sure how to troubleshoot this without more details. Are the bulbs all in the same subnet (and discoverable through UDP broadcast?)

**Comment by altercation** [comment]
@kpsuperplane I'm having a similar issue. Details:

1. Initial config of homebridge-wiz-lan worked well and bulbs could be controlled fine ( posted a demo even! https://twitter.com/ethanschoonover/status/1295200829394231297?s=20 )

2. Today I restarted Homebridge for system maintenance and the lights stopped working via this plugin

3. Testing via CLI works fine. E.g.:

`echo '{"method":"setPilot","env":"pro","params":{"mac":"FF:BB:29:D3:89:EE","src":"udp","state":true,"temp":2869,"dimming":100}}' | nc -u 192.168.90.72 38899`

4. Testing via Wiz app works fine

5. I uninstalled the homebridge-wiz-lan plugin and removed the plugin section from the config in Homebridge. Additionally I cleared cached accessories.

6. After a Homebridge restart I re-added the plugin and it does not scan or detect any accessories.

7. Other plugins (e.g. Nest) in Homebridge continue to function properly.

If there is anything else I can test or try, I'd be happy to. Thanks! Loved the plugin when it was working!

**Comment by iRayanKhan** [comment]
Hey @kpsuperplane I just installed this plugin with a bulb on the same network as my Pi, and it doesn't seem to pickup my RGB bulb. Is there a command I can use to ping it for the plugin to pickup?

I am able to control it locally and see it on my Wi-Fi network.

**Comment by code-in-progress** [comment]
@kpsuperplane All my bulbs are in a separate vlan (192.168.107.0) and my HB server is multi-honed (192.168.20.0 and 192.168.107.0) and it's also not picking up any bulbs. I can see the discovery going out through debug, but nothing ever comes back to the HB server.

Any ideas?

**Comment by kpsuperplane** [comment]
Hey everyone,

Unfortunately I don't know what might be causing your issues. I also don't have any bulbs anymore to debug with as I've transitioned my home to the Hue ecosystem.

My best suggestion is to look into setting up homeassistant instead; they have an actively supported wiz implementation https://github.com/sbidy/wiz_light

**Comment by code-in-progress** [comment]
> Unfortunately I don't know what might be causing your issues. I also don't have any bulbs anymore to debug with as I've transitioned my home to the Hue ecosystem.

LOL, I went the opposite direction: Hue > Wiz.

> My best suggestion is to look into setting up homeassistant instead; they have an actively supported wiz implementation https://github.com/sbidy/wiz_light

I'm currently using that and it's actually really well done. I wanted to have my bulbs (40+ of them) in both HB and HA. The main reason is that with iOS14, Homekit seems to have some issues with HA's HomeKit bridge.

Thanks for chiming in @kpsuperplane. Much appreciated.

**Comment by mitch7391** [comment]
@code-in-progress slightly off topic here, but given you have so many wiz bulbs. Do you have any issues with them emitting high frequency noises (quiet little beeps and chirps)? Iv taken two back due to the issue and now one that didn’t do it is starting to. It’s actually making want to move away from wiz; however, I really love the design of their large filament style bulbs.

**Comment by iRayanKhan** [comment]
So it turns out I'm just stupid and ran this plugin on the wrong Pi (diff network). I've since fixed it and 2 of my RBG bulbs work great in HomeKit. Thanks for the plugin!

**Comment by code-in-progress** [comment]
> @code-in-progress slightly off topic here, but given you have so many wiz bulbs. Do you have any issues with them emitting high frequency noises (quiet little beeps and chirps)? Iv taken two back due to the issue and now one that didn’t do it is starting to. It’s actually making want to move away from wiz; however, I really love the design of their large filament style bulbs.

Yeah, the filament bulbs tend to do that. The color bulbs (which I have the most of), don't, but if you have a LOT of broadcast UDP traffic, commands sometimes seem to just vanish into thin air. The other thing I've noticed is that the new A21 bulbs (https://www.homedepot.com/p/Philips-Color-and-Tunable-White-A21-LED-100-Watt-Equivalent-Dimmable-Smart-Wi-Fi-Wiz-Connected-Wireless-LED-Light-Bulb-562405/313643239) seem to die a lot. Might just have been a bad batch, but I got 5 of them and 2 of them have died in the past week. :(

**Comment by mitch7391** [comment]
Thanks for the update :) I will have to keep an eye on the filament bulbs then. Wiring hopefully wouldn’t be an issue as they were newly wired bedside pendant lights that I put these straight into (also the house is only 4 years old). The high pitch noise is so noticeable for me as it’s right next to me at night when everything else is quiet haha...

**Comment by code-in-progress** [comment]
> The high pitch noise is so noticeable for me as it’s right next to me at night when everything else is quiet haha...

Are these in some sort of a dimmer or on a circuit with a dimmer type switch? That's the only other thing I can think of that would cause that (especially since you already replaced the bulbs). Does it happen in the same fixture or on the same circuit? If you throw another bulb in there (even one like a cheap, dumb LED bulb), does it still happen?

I would be ripping my hair out if it was that noticeable, so I can only imagine what it's doing to you. lol

**Comment by mitch7391** [comment]
No dimmer switch, just a regular switch; the idea was that I knew I was going to have a smart bulb of some kind and that would handle the dimming. I might try another brand of bulb (I have some regular Hue laying around) to see if that happens, but I do know the one of my missus's side is not doing it and mine has done it twice (replacement didn't at first and does now when on, but the original was doing it while on and off).

It took me a week to realise it wasn't crickets in the house that I could faintly hear somewhere lol and eventually realised it only started once I got the lights installed, then I put my ear up against the bulb and could hear it very loudly... So yes, I was going crazy haha...

**Comment by Martynet** [comment]
Hi everyone, I had similar issue as above... Basically I installed Wiz plugin and the Light appeared in Homekit immediately and was working fine... But there was a "crack" in the light and seller sent me a replacement. After replacing the light, new one popped in my Homekit automatically and worked perfectly. But I wasn't sure how to remove the old one, as there was no option to remove it in Homekit app. Now I know that I have to remove it in Accessories section in Homebridge web page, but I didn't know that before... And I'm not sure exactly what I did, but both lights are gone now. I tried everything to bring it back, but no luck. I even deleted the Wiz app on my phone and completely re-installed whole Homebridge on my Synology NAS. But still no luck. Ring doorbell is working fine... The light works fine in Wiz app. It works in both Google and Alexa apps... The only place it doesn't work is Homebridge. It's making me crazy now :) If anyone can help, I'd really appreciate it. Thank you, Martin
